app/assets/javascripts/driver_form.js

```diff
@@ -25,7 +25,7 @@ function registerForm(id) {
                 if (data["error"] == null) {
                     if (data["saved"] != null) {
                         getDriverForm(id);
-                        $(".mentor_" + id + " > td:nth-child(4)").each(function(i){
+                        $(".mentor_" + id + " > td:nth-child(5)").each(function(i){
```

Comment:
Using `nth-child` here is really ugly and will likely cause errors in the future because we won't think that the JS is based on column count. Use a class name!